.github/workflows/test_python.yml

```diff
@@ -0,0 +1,29 @@
+name: test nodl
```

Comment:
Shall we rename this file to "test.yml" to follow the example of multi-project repos like rosbag2?